src/elements/core/styles/a11y.scss

```diff
@@ -3,3 +3,17 @@
 .sbb-screen-reader-only {
   @include a11y.screen-reader-only;
 }
+
+.sbb-focus-outline:focus-visible {
+  @include a11y.focus-outline;
+
+  border-radius: var(--sbb-border-radius-1x);
```

Comment:
I think we should not include border-radius as it can wrongly override consumers one.